R/run.R

```diff
@@ -477,8 +478,7 @@ copy_resources_implicit <- function(src, dst, resources, artefacts) {
   }
 
   copy_files(file.path(src, to_copy),
-             file.path(dst, to_copy),
-             overwrite = TRUE)
+             file.path(dst, to_copy))
```

Comment:
This function only ever gets called with a completely empty `dst` directory. There is no reason to use `overwrite = TRUE`.

Reply:
I'm sorry I don't really understand the comment. The only "previous call" in this file also omits the `make_writable` argument. `make_writable = FALSE` is a no-op, it is not going to make things read-only (not that we would want that here anyway).

Maybe having `copy_files` do the chmod is confusing, and instead I remove the `make_writable` argument and move the chmod to the only two places it is actually needed.

Reply:
previous call was outpack_store.R:38 where you have `make_writable = FALSE`, but it looks like that should be TRUE :)